File: Two-Sum.py
1class Solution:
2    def twoSum(self, nums: List[int], target: int) -> List[int]:
3        var_dict = dict()
4
5        for i, e in enumerate(nums):
6            search = target - e
7
8            if search in set(nums[i+1:]):
9                return [i, nums.index(search, i+1)]

File: Two-Sum.rs
1impl Solution {
2    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
3        for i in 0..nums.len() {
4            for j in (i + 1)..nums.len() {
5                if nums[i] + nums[j] == target {
6                    return vec![i as i32, j as i32];
7                }
8            }
9        }
10        vec![]
11    }
12}